src/Common/src/TypeSystem/IL/Stubs/PInvokeILEmitter.cs

```diff
@@ -20,473 +23,42 @@ namespace Internal.IL.Stubs
     /// </summary>
     public struct PInvokeILEmitter
     {
-        private MethodDesc _targetMethod;
-        private PInvokeILEmitterConfiguration _pinvokeILEmitterConfiguration;
-        private TypeSystemContext _context;
-        private PInvokeMetadata _importMetadata;
+        private PInvokeMethodData PInvokeMethodData;
```

Comment:
We follow https://github.com/dotnet/corefx/blob/master/Documentation/coding-guidelines/coding-style.md :

`_camelCase` for internal and private fields and use readonly where possible